spellcheck/platform/linux/spellcheck_linux.cpp

```diff
@@ -1,38 +1,117 @@
 // This file is part of Desktop App Toolkit,
 // a set of libraries for developing nice desktop applications.
 //
-// For license and copyright information please follow this link:
-// https://github.com/desktop-app/legal/blob/master/LEGAL
-//
+// Author: Nicholas Guriev <guriev-ns@ya.ru>, public domain, 2019
+// License: CC0, https://creativecommons.org/publicdomain/zero/1.0/legalcode
+
+#include <set>
+#include "spellcheck/spellcheck_utils.h"
+#include "spellcheck/platform/linux/enchant.h"
+
 #include "spellcheck/platform/linux/spellcheck_linux.h"
 
-namespace Platform::Spellchecker {
```

Comment:
Why you don't want to use `namespace Platform::Spellchecker`? =(

Reply:
Bjarne Stroustrup recommends the use of `Namespace::member` form for definitions instead of defining inside a namespace. (See his book "The C++ programming language", 3rd special edition; P. 8.4, advice No.7).

It makes sense because possible typos in function name would be discovered earlier, at compile time, not at linking.

Reply:
@mymedia2 While this seems true, I'm strongly against this approach right here, because it goes against the code style of all other modules in all desktop-app libraries and projects that use those. They all follow the same format in the .cpp files (except some very old):
```
#include-s

namespace Something::MaybeSomethingElse {
namespace {

// local definitions

} // namespace

// exported definitions

} // namespace Something::MaybeSomethingElse
```
And the motivation you mentioned isn't something that justify using a completely different approach in one single file.